2996_reflection/reflection.md

```diff
@@ -3696,66 +3840,60 @@ Extend the grammar for `$using-enum-declarator$` as follows:
   $using-enum-declarator$:
      $nested-name-specifier$@~_opt_~@ $identifier$
      $nested-name-specifier$@~_opt_~@ $simple-template-id$
-+    $splice-specifier$
++    $splice-type-specifier$
 ```
 :::
 
-Modify paragraph 1 of [enum.udecl]{.sref} as follows:
+Modify paragraph 1 to handle splicers:
 
 ::: std
 
-[1]{.pnum} A `$using-enum-declarator$` [that is not a `$splice-specifier$`]{.addu} names the set of declarations found by lookup ([basic.lookup.unqual]{.sref}, [basic.lookup.qual]{.sref}) for the `$using-enum-declarator$`. The `$using-enum-declarator$` shall designate a non-dependent type with a reachable `$enum-specifier$`.
+[1]{.pnum} [A `using-enum-declarator` of the form `$splice-type-specifier$` is considered in a type-only context, and designates the same construct designated by the splicer. Any other]{.addu} [A]{.rm} `$using-enum-declarator$` names the set of declarations found by type-only lookup ([basic.lookup.general]) for the `$using-enum-declarator$` ([basic.lookup.unqual], [basic.lookup.qual]). The `$using-enum-declarator$` shall designate a non-dependent type with a reachable `$enum-specifier$`.
```

Comment:
Missing $s around `using-enum-declarator`. Also "is a type-only context" rather than "is considered in a".

Also, shouldn't that part go into [temp] where we define what type-only context is? I think it'd be nice to have all those localized.

Reply:
It's kinda weird that `using-enum-declarator` _isn't_ listed in the `[temp.res.general]` list already? I'd almost rather leave it as-is, and learn during CWG review if there's a reason why.